src/components/Tool/Appeal/AppealsContext/AppealsContext.tsx

```diff
@@ -186,6 +186,7 @@ export const AppealsProvider: React.FC<AppealsContextProps> = ({
   // In the flows view, we need the total count of all contacts in every column, but the API
   // filters out contacts excluded from an appeal. We have to load excluded contacts also and
   // manually merge them with the other contacts.
+  const inFlowsView = !contactsFilters.appealStatus;
```

Comment:
Coudl you not check the viewMode against `TableViewModeEnum.Flows`?

Reply:
I was just blindly copying what was there, but that's a great suggestion!